app/models/spree/user_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::User.class_eval do
+  def full_name
```

Comment:
I could remove this `Spree::User` method in favor of defining a concern to be included into those decorators if that would be preferred? That would also clean up some of the repetition in those decorators.